.github/workflows/azure-deploy.yaml

```
@@ -19,14 +14,28 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
+      - name: Fetch docs
+        uses: actions/checkout@v3
+        with:
+          repository: "MaaAssistantArknights/MaaAssistantArknights"
+          sparse-checkout: "docs"
+          path: "temp"
+          ref: "master"
+          fetch-depth: "1"
+
+      - name: Move and cleanup docs
+        run: |
+          mv ./temp/docs ./apps
+          rm -r ./temp
+
       - name: Setup Node.js environment
         uses: actions/setup-node@v3
         with:
           node-version: 16
```

```suggestion
          node-version: 18
```

建议改成 node 18，node 16 的终止支持日期[已经被提前了](https://nodejs.org/en/blog/announcements/nodejs16-eol)